lib/open_telemetry/honeycomb/json.ex

```diff
@@ -5,18 +5,18 @@ defmodule OpenTelemetry.Honeycomb.Json do
   The OpenTelemetry Honeycomb Exporter uses a `Jason`-style JSON encoder via a behaviour so you
   can adapt it to your preferred JSON encoder or decode/encode options.
 
-  To use `Poison`, install it. The exporter defaults `json_backend` to
+  To use `Poison`, install it. The exporter defaults `json_module` to
   `OpenTelemetry.Honeycomb.Json.PoisonBackend`, which adapts `Poison` to the subset of the
   `Jason` API we document with this behaviour.
 
-  To use `Jason`, install it and configure it as the `json_backend`:
+  To use `Jason`, install it and configure it as the `json_module`:
 
   ```
   config :opentelemetry,
     processors: [
       otel_batch_processor: %{
-        exporter: OpenTelemetry.Honeycomb.Exporter,
-        json_backend: Jason
+        exporter:
+          {OpenTelemetry.Honeycomb.Exporter, json_module: Jason}
```

Comment:
I'll have to trust you on this one. Did it change in any particular version of upstream OT, or did I just leave it unchanged from the OC days?